src/DGtal/shapes/doc/moduleSurfaceMesh.dox

```diff
@@ -235,5 +235,76 @@ corresponding to relative distances 0.25, 0.5 and 0.75.
 
 @image html spot-bft-isolines.png "SurfaceMesh faces colored according to distance to bluest vertex and three isodistance lines"
 
+\section Flipping edges
+
+Since 1.4, you may modify a SurfaceMesh through flips. Only edges that
+are incident to a triangle on the right and a triangle on the left are
```

Comment:
```suggestion
are adjacent to two triangles are
```

Reply:
Oh ok from the code you mean left/right with respect to the fact that the edge is oriented.

Not sure that it matters here (for sure, internally, you make sure that the surface is still consistent enough terms of face orientation).

Reply:
At least it is required in the code at the moment. Obviously it prevents using flips on unoriented surfaces like a Möbius band. However I am not sure that the whole code would work for such surfaces as it is written now. So I prefer to leave the documentation as is here.

Reply:
Ok. Done

Reply:
👌 thx